src/tests/math/math_test_fp.c

```diff
@@ -0,0 +1,89 @@
+/**
+ * @math_test_fp
+ * @simple tests for fp functions in embox just to make sure the corresponding functions are not stubs
+ * @date 03.03.2020
+ * @author Rushil kvs
+ */
+
+#include <embox/test.h>
+#include <math.h>
+
+EMBOX_TEST_SUITE("Tests for fp functions");
+
+bool Equal(double a, double b)
+{
```

Comment:
Can you please follow our code style? https://github.com/embox/embox/wiki/Code-Style

We are using lower case functions namings.

Reply:
Thanks for reviewing, I'll do the required changes